base_local_planner/src/latched_stop_rotate_controller.cpp

```diff
@@ -135,6 +136,13 @@ bool LatchedStopRotateController::stopWithAccLimits(const tf::Stamped<tf::Pose>&
     boost::function<bool (Eigen::Vector3f pos,
                           Eigen::Vector3f vel,
                           Eigen::Vector3f vel_samples)> obstacle_check) {
+  if (stopping_scaling_percent_ <= 0.0) {
```

Comment:
Can this percentage be negative? from GRM config I am not sure what is the range for this..

Reply:
It shouldn't be, 0 is allowed and this was the "lets skip unnecessary computation" clause, but figured a safety check is fine, incase someone tries it.

Reply:
yeah.. especially when we are trying to multiply this value later to velocities. There might be some way in the GRM config to put a range.. or may be setting it as an unsigned int.